packages/aws-client/tests/test-StepFunctions.js

```diff
@@ -7,31 +7,16 @@ const awsServices = require('../services');
 const { throttleOnce } = require('../test-utils');
 const StepFunctions = require('../StepFunctions');
 
-const runWithStubbedAndThrottledSfnOperation = async (operation, response, fn) => {
-  const operationBefore = awsServices.sfn()[operation];
-  try {
-    const promise = throttleOnce(() => Promise.resolve(response));
-
-    awsServices.sfn()[operation] = () => ({ promise });
-
-    return await fn();
-  } finally {
-    awsServices.sfn()[operation] = operationBefore;
-  }
-};
```

Comment:
Don't need this? What replaces it? 

(Likely about to answer my own question reading on)

Reply:
Is it just that we're more easily able to sub AWS functions? Don't need the throttling here?

Reply:
Well, the tests that use this runWithStubbedAndThrottledSfnOperation function were updated to throttle inside each test, so this function became unused, and was removed.